pkgs/development/embedded/arduino/arduino-language-server/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, buildGoModule, fetchFromGitHub}:
+
+
+buildGoModule rec {
+  pname = "arduino-language-server";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "arduino";
+    repo = pname;
```

Comment:
While widely used, `repo` and `pname` semantically are not equal. For example, we can change `pname` of derivation with `overrideAttrs`, but in such case `repo` will be invalid.